logstash/values.yaml

```diff
@@ -150,7 +150,7 @@ livenessProbe:
   httpGet:
     path: /
     port: http
-  initialDelaySeconds: 60
+  initialDelaySeconds: 90
```

Comment:
I think this is still cutting it pretty close and could make it hard for Logstash processes that are a bit slower to startup. This should be high enough that we can be very sure that it isn't going to prevent a container from starting up. Maybe around 5 minutes would be a nicer value.